.gitignore

```diff
@@ -5,3 +5,4 @@ lib/**/*
 **/.env
 **/env
 **/venv
+**/.DS_Store
```

Comment:
This line is redundant.  `.DS_Store` files are already ignored by the parent directory wildcard `**`.